BLE Indoor Positioning/src/test/java/com/nexenio/bleindoorpositioning/location/distance/BeaconDistanceCalculatorTest.java

```diff
@@ -31,4 +58,18 @@ public void calculateDistance_calibratedRssi_calibratedDistance() throws Excepti
         assertEquals(1, calculatedDistance, 0.1);
     }
 
+    public Beacon createDummyBeacon() {
+        Beacon dummyBeacon = new IBeacon();
+        dummyBeacon.setLocationProvider(new LocationProvider() {
+            @Override
+            public Location getLocation() {
+                Location location = new Location(LocationTest.BERLIN);
+                location.setAltitude(2);
```

Comment:
Test with different altitudes but equal locations and different locations with steady altitude.

Reply:
The distance here is calculated to one beacon and not between beacons